coderd/database/queries/workspaceagentstats.sql

```diff
@@ -330,7 +330,9 @@ WITH agent_stats AS (
 		coalesce(SUM(session_count_reconnecting_pty), 0)::bigint AS session_count_reconnecting_pty,
 		coalesce(SUM(connection_count), 0)::bigint AS connection_count
 	FROM workspace_agent_stats
-	WHERE usage = true
+	-- We only want the latest stats, but those stats might be
+	-- spread across multiple rows.
+	WHERE usage = true AND created_at > now() - '1 minute'::interval
```

Comment:
This query, `GetWorkspaceAgentUsageStatsAndLabels` is called with the time the server last started, in `prometheusmetrics.go` (`initialCreateAfter) - but with the experiment enabled, stats from clients are reported at most every minute.